### PR TITLE
Fix CI for example 8

### PR DESCRIPTION
**Pull request by jaimergp**
CI is working, but the code behind #94 is not!

### CODE REVIEW

**Comment by jaimergp** [comment]
I don't know what's wrong. The listeners do not seem to intercept the streams at all :/

**Comment by goanpeca** [comment]
@jaimergp I think you need to do the build step so that the assets get compiled and then that is what is used by actions. 

Are you running
```
npm run format
npm run check
npm run build
```

**Comment by jaimergp** [comment]
🤦  I thought that was being done on the precommit hook because it took a bit longer than usual. Heh 😬

**Comment by goanpeca** [comment]
It works :) !

File: .github/workflows/example-8.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
           environment-file: etc/example-faulty-environment.yml
       - name: Check previous step failed
         shell: bash
-        run: ${{ steps.setup-miniconda.conclusion == "failure" }}
+        run: ${{ steps.setup-miniconda.conclusion == 'failure' }}
```
Comment (goanpeca) [inline .github/workflows/example-8.yml:30]: So since the previous step fails, this will never be reached.

Comment (goanpeca) [reply .github/workflows/example-8.yml:31]: https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions

Comment (jaimergp) [reply .github/workflows/example-8.yml:31]: Hm, I fixed that and it runs now but the CI result is still red. I thought last successful one counted but maybe we have to setup that explicitly?

**Comment by jaimergp** [comment]
Ok, finally found the right combo. To check for expected failures:

* The step that is expected to fail must have `continue-on-error: true` and an `id`.
* The evaluation step needs `if: always()` and it should check for `outcome == 'failure'` not `conclusion == 'failure'`.

This is ready now!

**Comment by goanpeca** [comment]
>Ok, finally found the right combo. To check for expected failures:

Awesome work @jaimergp. Thanks a lot!

**Pull request merged**